admin/module/tests/_support/DatabaseTestCase.php

```diff
@@ -1,7 +1,12 @@
 <?php namespace Tests\Support;
 
-class DatabaseTestCase extends \CodeIgniter\Test\CIDatabaseTestCase
+use CodeIgniter\Test\CIUnitTestCase;
+use CodeIgniter\Test\DatabaseTestTrait;
+
+class DatabaseTestCase extends \CodeIgniter\Test\CIUnitTestCase
```

Comment:
It looks like you got most of these but missed this one. 
```suggestion
class DatabaseTestCase extends CIUnitTestCase
```